smartseq2_single_sample/tasks/Kallisto.wdl

```diff
@@ -0,0 +1,219 @@
+
```

Comment:
My tasks are all in pipelines/tasks -- which is supposed to be our generic pipelines task folder. Was it necessary to duplicate the WDL here? I think it would be better to pull from the original task and modify that one if you needed to change it.

Reply:
I just try to avoid conflict as much as possible now. I admit we should put them together later